config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /en|vi/ do
-    get  "sessions/new"
+    get "sessions/new"
```

Comment:
cần dòng này nữa không? chị thấy có `get "/login", to: "sessions#new"` rồi mà
check lại indent dòng 7-10 nhé

Reply:
indent hình như lên git nó bị lệch ạ, bên local em thấy nó thế này ạ
![screenshot from 2018-12-26 09-12-45](https://user-images.githubusercontent.com/45023665/50429094-cd3bd580-08ee-11e9-99b8-9467754b7046.png)